backend/editor/entries.py

```diff
@@ -244,10 +244,13 @@ def close_project(self):
     def list_existing_projects(self):
         """
         Helper function for listing all existing projects created in Taxonomy Editor
+        includes number of nodes with label ERROR for each project
         """
         query = """
             MATCH (n:PROJECT)
-            WHERE n.status = "OPEN" RETURN n
+            OPTIONAL MATCH (error_node:ERRORS {{branch_name: n.branch_name, id: n.id}})
+            WHERE n.status = "OPEN"
+            RETURN n, size(error_node.errors) AS error_count
             ORDER BY n.created_at
         """
         result = get_current_transaction().run(query)

```

Comment:
Here we should transform the result to include our error count in result:

```suggestion
        result = get_current_transaction().run(query)
        # pack errors count in result
        result = [dict(node, errors_count=num_errors) for node, num_errors in result]

```

Reply:
Ah, great! I was trying to do this in the query itself 😅